percy/lib/cli_wrapper.py

```diff
@@ -42,8 +42,12 @@ def is_percy_enabled():
             log(e, on_debug=True)
             return False
 
-    def post_screenshots(self, name, tag, tiles, external_debug_url=None, ignored_elements_data=None, considered_elements_data=None, sync=None):
-        body = self._request_body(name, tag, tiles, external_debug_url, ignored_elements_data, considered_elements_data, sync)
+    def post_screenshots(self, name, tag, tiles, external_debug_url=None,
+        ignored_elements_data=None, considered_elements_data=None, sync=None, test_case=None, th_execution_id=None
```

Comment:
above we are using `th_test_case_execution_id`  and here `th_execution_id`, lets keep this consistent

Reply:
addressed. Did that to keep name short.